app/models/user.py

```diff
@@ -0,0 +1,20 @@
+from sqlmodel import SQLModel, Field
+from sqlalchemy import UniqueConstraint
+
+from app.models.util import TimestampModel, UUIDModel
+
+
+class UserBase(SQLModel):
+    email: str = Field(max_length=255, nullable=False)
+    full_name: str = Field(max_length=255, nullable=False)
+
+
+class User(UUIDModel, TimestampModel, UserBase, table=True):
+    __tablename__ = "Users"
+    __table_args__ = (UniqueConstraint("email"),)
```

Comment:
Why do we need a UUID if email is both mandatory and constrained to be unique?

Reply:
Main thought was to show use of UUIDModel as mixin for primary key everywhere. As for the unique constraint, it was attached to email as I wanted to see what happened if the db raised an error due to violating a db constraint and how it was/should be handled at the different layers